src/ui/Languages/el-GR.xml

```diff
@@ -6,7 +6,7 @@
     <TranslatedBy>Μετάφραση: Παναγιώτης Μητσάκης</TranslatedBy>
     <CultureName>el-GR</CultureName>
     <HelpFile />
-    <Ok>&amp;Εντάξει</Ok>

```

Comment:
This will remove the "alt+E" shortcut - is that on purpose?

Reply:
No. I will fix it.